### PR TITLE
feat/update: further improvements to uiux

### PR DESCRIPTION
**Pull request by DannyS03**
## Context

Continuation of https://github.com/maticnetwork/matic-docs/pull/1376

- Improves card styling
- Segments and adds background to text blobs between sections
- Updates all logos to the latest (available)
- Enhances navbar (temporarily removes contribute dropdown)
- Revises the approach to the use card row until there is sufficient content available for use cases.
- General edits

### CODE REVIEW

File: docs/supernets/operate/local-blockchain.md [hunk 1]
```diff
@@ -0,0 +1,259 @@
+---
+id: supernets-local-deploy
+title: Create a local blockchain
+sidebar_label: Create a local blockchain
+description: "An introduction to Polygon Supernets."
```
Comment (ashucoder9) [inline docs/supernets/operate/local-blockchain.md:5]:
```suggestion
description: Create a local blockchain using Polygon Supernets.
```

File: docs/supernets/operate/local-blockchain.md [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+## Prerequisites
+
+Before starting this tutorial, you should understand the basics of blockchain technology and have some familiarity with command-line interfaces. You should also have the polygon-edge binary installed on your machine. If you haven't already, check out the [installation guide](install.md) for more information.
+
+## What you'll do
+
+The tutorial will cover the following steps:
+
+1. Initialize PolyBFT consensus.
+2. Configure bootnodes
```
Comment (ashucoder9) [inline docs/supernets/operate/local-blockchain.md:124]:
```suggestion
2. Configure bootnodes.
```

**Comment by DannyS03** [comment]
Oh shoot didn't mean to commit the supernet docs

**Comment by ashucoder9** [comment]
We can keep them as they won't be published 🙂

**Comment by DannyS03** [comment]
I removed them as they are drafts (also these are an older versions anyways).

**Review by ashucoder9 — APPROVED** [review]
Awesome work @DannyS03 🫡🙏🏼

**Pull request merged**